resources/lib/library.py

```diff
@@ -1494,8 +1494,8 @@ def explorer( self, history, location, label, thumbnail, itemType, isWidget = Fa
                     listitem.setProperty( "widgetPath", location )
 
                 elif itemType == "32011" or itemType == "32019" or itemType == "32073":
-                    action = 'ActivateWindow(10501,"' + location + '",return)'
-                    listitem.setProperty( "windowID", "10501" )
+                    action = 'ActivateWindow(10502,"' + location + '",return)'
+                    listitem.setProperty( "windowID", "10502" )
```

Comment:
Does it make a practical difference? The decision to use window id's for add-ons is because the script doesn't apply any visibility conditions to ActivateWindow([windowid],...) whereas it does to the alias - and with this fix music add-ons wouldn't show unless there is music scanned into the library/the visibility conditions are made more complex...